pubsub/topics/update_topic_type.go

```diff
@@ -38,18 +40,25 @@ func updateTopicType(w io.Writer, projectID, topicID string) error {
 	}
 	defer client.Close()
 
-	updateCfg := pubsub.TopicConfigToUpdate{
-		// If wanting to clear ingestion settings, set this to zero value: &pubsub.IngestionDataSourceSettings{}
-		IngestionDataSourceSettings: &pubsub.IngestionDataSourceSettings{
-			Source: &pubsub.IngestionDataSourceAWSKinesis{
-				StreamARN:         streamARN,
-				ConsumerARN:       consumerARN,
-				AWSRoleARN:        awsRoleARN,
-				GCPServiceAccount: gcpServiceAccount,
+	updateReq := &pubsubpb.UpdateTopicRequest{
+		Topic: &pubsubpb.Topic{
```

Comment:
nit: declare and instantiate the `Topic` outside of the request object. The object declaration nesting here starts to get hard to read with the line wrap.